src/ec/packages/ec-component-social-media-follow/social-media-follow.html.twig

```diff
@@ -0,0 +1,77 @@
+{#
+  Parameters:
+    - "description" (string) (default: '')
+    - "variant" (string) (default: ''): can be 'vertical'
+    - "links" (array) (default: []): format: [
+        {
+          "path" (string) (default: ''),
+          "label" (string) (default: '')
+          "icons" (array) : format: [
+            {
+              "name" (string),
+              "size": (string),
+              "path": (string),
+            },
+            ...
+          ]
+        },
+        ...
+      ]    
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-social-media-follow' %}
+{% set _extra_attributes = '' %}
+{% set _description = description|default('') %}
+{% set _links = links|default({}) %}
+{% set _variant = variant|default('') %}
+
+{# Internal logic - Process properties #}
+
+{% if _variant is defined and _variant is not empty %}
+  {% set _css_class = _css_class ~ ' ecl-social-media-follow--' ~ _variant %}
+{% endif %}  
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+<div class="{{ _css_class }}"{{ _extra_attributes|raw }}>
```

Comment:
please wrap the printed content between `{% spaceless %}` and `{% endspaceless %}`

also every `{{` and `{%` should have an additional `-` (`{{-`, `{%-`, and the same for closing ones). See http://vvv.tobiassjosten.net/symfony/trim-and-remove-twig-whitespace/